ptn/boozebot/botcommands/DatabaseInteraction.py

```diff
@@ -155,7 +166,7 @@ async def _reconfigure_workbook_and_form(self):
             # Update the tracking sheet object
             self.tracking_sheet = await workbook.get_worksheet(self.worksheet_with_data_id)
         except gspread.exceptions.APIError as e:
-            print(f"Error reading the worksheet: {e}")
+            logging.error(f"Error reading the worksheet: {e}")
```

Comment:
```suggestion
            logging.exception(f"Error reading the worksheet: {e}")
```

Since we use logging anyway, might as well get the traceback. Same below?